README.md

```diff
@@ -74,87 +41,30 @@ Add a new file called `vcpkg.json` on the root of your CMake project and add the
+project(HelloWorld)
 
-```bash
-# Create a build folder (if there's not one already there)
-mkdir build
-cd build
-cmake ..
-cmake --build .
-```
-
-> Using Vcpkg manifest makes easy to define multiple dependencies and delegate building them to Vcpkg.
-
-#### CMake Project + fetch content
-
-For this scenario, CMake will fetch the Azure SDK source code and make it part of your project. The SDK client libraries will be compiled at the same time as your application.
-
-Follow the step 1 from above to create a CMake project first.
-
-##### 2. Define CMake fetch content
-
-Add the following code to your root `CMakeLists.txt` file:
-
-```cmake
-# Add this code before creating and linking your application
-
-include(FetchContent)
-FetchContent_Declare(
-    azuresdk
-    GIT_REPOSITORY      https://github.com/Azure/azure-sdk-for-cpp.git
-    GIT_TAG             <Release Tag or Git-Commit-Here>
-)
-FetchContent_GetProperties(azuresdk)
-if(NOT azuresdk_POPULATED)
-    FetchContent_Populate(azuresdk)
-    # Adding all Azure SDK libraries
-    add_subdirectory(${azuresdk_SOURCE_DIR} ${azuresdk_BINARY_DIR} EXCLUDE_FROM_ALL)
-    # Adding one Azure SDK Library only (Storage blobs)
-    # add_subdirectory(${azuresdk_SOURCE_DIR}/sdk/storage/azure-storage-blobs ${azuresdk_BINARY_DIR} EXCLUDE_FROM_ALL)
-endif()
-```
-
-##### 3. Link Azure SDK libraries to your application
+find_package(azure-identity-cpp CONFIG REQUIRED)
+find_package(azure-storage-blobs-cpp CONFIG REQUIRED)
 
-The only difference from the previous scenario is that you don't need to call `find_package()`, since the cmake targets are integrated to your project. So you only need:
+add_executable(HelloWorld helloworld.cpp)
 
-```cmake
-# After creating the cmake target
-target_link_libraries(quick-sample PRIVATE Azure::azure-identity Azure::azure-storage-blobs)
+target_link_libraries(HelloWorld PRIVATE Azure::azure-identity Azure::azure-storage-blobs)
 ```
 
-> Note: You need to take care of getting the Azure SDK dependencies on your own. Either manually installing them or by integrating the source code to your project as well.
-
-Use step 5 from previous scenario to generate and build your project.
-
-> This scenario requires extra manual configuration to get dependencies, but it is useful as an alternative when Vcpkg is not available
-
-#### Other combinations
+### Additional methods for installing and configuring
 
-It should be possible to create your application without a CMake project. For example, manually cloning Azure SDK, building libraries and finally linking them to your application. However, this is considered an advanced scenario and it is not either described or maintained (The other scenarios described below are validated with CI pipelines).
+- [Get started by cloning a sample project](https://github.com/antkmsft/az-helloworld-cpp)
```

Comment:
probably should use a project in a neutral environment, rather than Anton's private github instance.

Reply:
Agreed. I'll look into what we need to do to create another repo in the azure org or if there is a better place to create it.